awscli/customizations/configure/configure.py

```diff
@@ -124,6 +124,7 @@ def _run_main(self, parsed_args, parsed_globals):
                 section = profile_to_section(profile)
                 new_values['__section__'] = section
             self._config_writer.update_config(new_values, config_filename)
+        return 0
```

Comment:
For the commands that subclass from `BasicCommand` what is your thoughts on adding logic in the `main()` that if `_main()` does not return anything, the command will return a successful rc on your behalf? I'm mainly suggesting it because it is not necessarily obvious and can be easy to forget when writing these commands that an rc needs to be returned.